### ISSUE TITLE
Lenovo X1 2018 touch screen support

### ISSUE THREAD

**Issue by tec** [user501254/TouchpadIndicator#30]
Please add support for Lenovo X1 2018 touch screen:

XInput identifies touch screen as "Raydium Corporation Raydium Touch System".

I suggest to add 'touch system' to lib.js const TOUCHSCREENS:
https://github.com/user501254/TouchpadIndicator/blob/de79bd556b2ea520bdbb6d032a26f916987f9c3e/lib.js#L60

```javascript
const TOUCHSCREENS = Array('touchscreen', 'maxtouch', 'touch digitizer', 'touch system');
```

Thanks

**Comment by user501254** [comment]
Hi @tec 
Have you tested this on your system? I would be happy to merge a PR if you can make one.

**Comment by tec** [comment]
Yes, I tested it on my X1 Thinkpad.
Before the fix the third option was missing in the indicator menu:

![touchpadindicator with touchscreen](https://user-images.githubusercontent.com/3347/46630652-13319a80-cb45-11e8-9ee7-d580d6ee9500.png)

I'll try to create a PR.

**Comment by user501254** [comment]
Merged 7e98267dfef5c75e5f429fe69dc17a91829aa7f1.

**Comment by tec** [comment]
Yay, thank you!
